tsl/test/expected/continuous_aggs_multi.out

```diff
@@ -0,0 +1,172 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+-- stop the continous aggregate background workers from interfering
+\c :TEST_DBNAME :ROLE_SUPERUSER
+-- stop the continous aggregate background workers from interfering
+SELECT _timescaledb_internal.stop_background_workers();
+ stop_background_workers 
+-------------------------
+ t
+(1 row)
+
+\c :TEST_DBNAME :ROLE_DEFAULT_PERM_USER
+CREATE TABLE continuous_agg_test(timeval integer, col1 integer, col2 integer);
+select create_hypertable('continuous_agg_test', 'timeval', chunk_time_interval=> 2);
+NOTICE:  adding not-null constraint to column "timeval"
+        create_hypertable         
+----------------------------------
+ (1,public,continuous_agg_test,t)
+(1 row)
+
+INSERT INTO continuous_agg_test VALUES
+    (10, - 4, 1), (11, - 3, 5), (12, - 3, 7), (13, - 3, 9), (14,-4, 11),
+    (15, -4, 22), (16, -4, 23);
+-- TEST for multiple continuous aggs 
+--- invalidations are picked up by both caggs
+CREATE VIEW cagg_1( timed, cnt ) 
+WITH ( timescaledb.continuous , timescaledb.refresh_lag = '-2')
+AS
+    SELECT time_bucket( 2, timeval), COUNT(col1) 
+    FROM continuous_agg_test
+    GROUP BY 1;
+CREATE VIEW cagg_2( timed, grp, maxval) 
+WITH ( timescaledb.continuous, timescaledb.refresh_lag = '-2' )
+AS
+    SELECT time_bucket(2, timeval), col1, max(col2) 
+    FROM continuous_agg_test
+    GROUP BY 1, 2;
+NOTICE:  adding index _materialized_hypertable_3_grp_timed_idx ON _timescaledb_internal._materialized_hypertable_3 USING BTREE(grp, timed)
+select view_name, view_owner, refresh_lag, refresh_interval, max_interval_per_job , materialization_hypertable 
+from timescaledb_information.continuous_aggregates;
+ view_name |    view_owner     | refresh_lag | refresh_interval | max_interval_per_job |            materialization_hypertable            
+-----------+-------------------+-------------+------------------+----------------------+--------------------------------------------------
+ cagg_1    | default_perm_user | -2          | @ 12 hours       | 40                   | _timescaledb_internal._materialized_hypertable_2
+ cagg_2    | default_perm_user | -2          | @ 12 hours       | 40                   | _timescaledb_internal._materialized_hypertable_3
+(2 rows)
+
+refresh materialized view cagg_1;
+INFO:  new materialization range for public.continuous_agg_test (time column timeval) (18)
+INFO:  materializing continuous aggregate public.cagg_1: new range up to 18
+select * from cagg_1 order by 1;
+ timed | cnt 
+-------+-----
+    10 |   2
+    12 |   2
+    14 |   2
+    16 |   1
+(4 rows)
+
+SELECT time_bucket(2, timeval), COUNT(col1) as value
+FROM continuous_agg_test
+GROUP BY 1 order by 1;
+ time_bucket | value 
+-------------+-------
+          10 |     2
+          12 |     2
+          14 |     2
+          16 |     1
+(4 rows)
+
```

Comment:
Check that `cagg_2` wasn't materialized? (the later one only checks invalidations, not new data)

Reply:
added